test/nightly/deploy.test.ts

```diff
@@ -44,42 +41,45 @@ interface IParallelTest {
 }
 
-suite('Create Web App and deploy', function (this: Mocha.Suite): void {
+suite.only('Create Web App and deploy', function (this: Mocha.Suite): void {
     this.timeout(6 * 60 * 1000);
     const testCases: ITestCase[] = [
         {
             runtimePrefix: 'Node',
             workspaceFolder: 'nodejs-docs-hello-world',
             versions: [
-                { version: '12', supportedAppOs: 'Both', displayText: '12 LTS' },
-                { version: '14', supportedAppOs: 'Both', displayText: '14 LTS' },
-                { version: '16', supportedAppOs: 'Both', displayText: '16 LTS' }
-            ]
-        },
-        {
-            runtimePrefix: 'Node',
-            workspaceFolder: 'testFolder',
-            zipFile: 'node-hello-1.zip',
-            versions: [
-                { version: '16', supportedAppOs: 'Both', displayText: '16 LTS' }
-            ]
-        },
-        {
-            runtimePrefix: '.NET',
-            workspaceFolder: undefined,
-            versions: [
-                { version: '3.1', supportedAppOs: 'Both', displayText: 'Core 3.1 (LTS)' },
-                { version: '5.0', supportedAppOs: 'Both', displayText: '5', buildMachineOsToSkip: 'darwin' }, // Not sure why this fails on mac build machines - worth investigating in the future
-                { version: '6.0', supportedAppOs: 'Both', displayText: '6 (LTS)', buildMachineOsToSkip: 'darwin' }
+                { version: '16', supportedAppOs: 'Both', displayText: '16 LTS' },
+                { version: '18', supportedAppOs: 'Both', displayText: '18 LTS' },
+                { version: '20', supportedAppOs: 'Both', displayText: '20 LTS' }
             ]
         },
+        // Todo: Fix / enable zipFile and .NET tests
+        // {
+        //     runtimePrefix: 'Node',
+        //     workspaceFolder: 'node-zip',
+        //     zipFile: 'node-hello-1.zip',
+        //     versions: [
+        //         { version: '20', supportedAppOs: 'Both', displayText: '20 LTS' }
+        //     ]
+        // },
+        // {
+        //     runtimePrefix: '.NET',
+        //     workspaceFolder: 'dotnet-hello-world',
+        //     versions: [
+        //         { version: '6', supportedAppOs: 'Both', displayText: '6 (LTS)' },
+        //         { version: '7', supportedAppOs: 'Both', displayText: '7 (STS)', /** buildMachineOsToSkip: 'darwin' */ }, // Not sure why this fails on mac build machines - worth investigating in the future
+        //         { version: '8', supportedAppOs: 'Both', displayText: '8 (LTS)', /** buildMachineOsToSkip: 'darwin' */ }
+        //     ]
+        // },
         {
             runtimePrefix: 'Python',
             workspaceFolder: 'python-docs-hello-world',
             versions: [
-                { version: '3.7', supportedAppOs: 'Linux' },
                 { version: '3.8', supportedAppOs: 'Linux' },
-                { version: '3.9', supportedAppOs: 'Linux' }
+                { version: '3.9', supportedAppOs: 'Linux' },
```

Comment:
Are any of these versions being deprecated? We probably don't need to test 5 different versions.

Reply:
I'm not sure about future deprecations, but these are all currently being offered in our create experience, so I figured I'd add them to match how it seems we were testing.  Are there any you'd prefer I remove?